src/BrAPINode.js

```diff
@@ -247,6 +247,10 @@ class BrAPICallController {
         else{
             url = url+BrAPICallController.formatURLParams(params)
         }
+        if (method == "patch") {
```

Comment:
since Brapi doesn't work with patch can we close this pull request?

Reply:
Hi @MFlores2021,
Yeah, that's fine for me.
I think we had it in our branch because we had some "unofficial" patch call at some point.